aries-site/src/examples/templates/page-layouts/anatomy/PageContainerWide.js

```diff
@@ -1,100 +1,64 @@
 import { useContext } from 'react';
-import { Box, Diagram, Stack, Text, ThemeContext } from 'grommet';
-import { LinkPrevious, LinkNext } from 'grommet-icons';
+import { Header, Main, ResponsiveContext } from 'grommet';
 
+import { AppContainer, PageContainer, pageContainer } from '../components';
 import { ContentArea } from './components';
 
-const connections = [
-  {
-    fromTarget: 'left-edge',
-    toTarget: 'label',
-    anchor: 'horizontal',
-    color: 'text-strong',
-  },
-  {
-    fromTarget: 'right-edge',
-    toTarget: 'label',
-    anchor: 'horizontal',
-    color: 'text-strong',
-  },
-];
-
-const PAGE_CONTAINER_WIDTH = '1608px';
-const PAGE_CONTAINER_SCALE = '80%';
+// `demoStyle` is specific for the Design System site and is used
+// as a visual aid to help present layout concepts. Remove from
+// your implementation.
+const demoStyle = {
+  border: { style: 'dashed' },
+  flex: true,
+  gap: 'small',
+  pad: 'small',
+  round: { size: 'xsmall' },
+};
 
 export const PageContainerWide = () => {
-  const theme = useContext(ThemeContext);
-  const diagramHeight = theme.global.size.medium;
-  const diagramWidth = `${(diagramHeight.replace('px', '') * 4) / 3}px`;
-  const annotationMargin = `${(100 - PAGE_CONTAINER_SCALE.replace('%', '')) /
-    2}%`;
-
-  const widthAnnnotation = (
-    <Box
-      direction="row"
-      align="center"
-      justify="between"
-      fill="vertical"
-      margin={{ horizontal: annotationMargin }}
-      pad={{ top: 'xlarge' }}
-    >
-      <LinkPrevious id="left-edge" color="text-strong" />
-      <Box id="label" pad={{ horizontal: 'xsmall' }} background="orange">
-        <Text color="text-strong" weight="bold">
-          {PAGE_CONTAINER_WIDTH}
-        </Text>
-      </Box>
-      <LinkNext id="right-edge" color="text-strong" />
-    </Box>
-  );
+  const size = useContext(ResponsiveContext);
+  /* Children of PageContainer should maintain horizontal whitespace on
+   * each side of the child so that the content has room to breathe and
+   * is not tightly condensed against the browser window's edge.
+   *
+   * Spacing is executed as the padding on each child to allow background
+   * colors and scroll regions to behave properly.
+   */
+  const containerPad = { horizontal: pageContainer.pad[size].horizontal };
```

Comment:
```suggestion
  const containerPad = { pageContainer.pad[size] };
```
does this work?

Reply:
This does work... until we add a vertical pad, then things go unexpectedly out of whack, but I think the risk is low and this is overprotective. I will simplify.

Reply:
Done.